### ISSUE TITLE
About the choice of training and testing datasets

### ISSUE THREAD

**Issue by jimo17** [machanic/SimulatorAttack#10]
Hi and thanks for the open source. I am very interested in your paper. I have some questions.
When you generate training data for the CIFAR-10 dataset by performing a Bandits attack under the norm, you mention 
> The data sources used by Bandits can be any image downloaded from the Internet. In this study, we use the training sets of the standard datasets with different data distributions from the tested images.

I wonder, by "different distributions", do you mean different datasets? Or is it a train and test set of the same dataset?
In other words, is the training data set generated by the Bandits attack in the experiments of your paper and the test data set used in the attack using the simulator, are they from the same data set?
